### PR TITLE
added documentation regarding recurreing event udpates and email noti…

### PR DESCRIPTION
**Pull request by baywet**
…fications

### CODE REVIEW

**Comment by baywet** [comment]
anyone? CC @Lauragra  @angelgolfer-ms

File: api-reference/beta/api/event_update.md [hunk 1]
```diff
@@ -59,8 +59,10 @@ In the request body, supply the values for relevant fields that should be update
...
 | subject|String|The text of the event's subject line.|
 
-Since the **event** resource supports [extensions](../../../concepts/extensibility_overview.md), you can use the `PATCH` operation to 
-add, update, or delete your own app-specific data in custom properties of an extension in an existing **event** instance.
+Because the **event** resource supports [extensions](../../../concepts/extensibility_overview.md), you can use the `PATCH` operation to 
+add, update, or delete your own app-specific data in custom properties of an extension in an existing **event** instance.  
+  
+If the **event** you're updating is the master event of a recurring series, contains multiple attendees, and has instances that have been updated separately, multiple notification emails will be sent out: one for the master series and one per instance that has been updated.
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/event_update.md]: Hi Vincent, thanks for your contribution! You mentioned in the "if" statement, the recurring meeting involves multiple attendees. Is that a required condition for the multiple notification emails to be sent out?

**Comment by baywet** [comment]
Maybe the statement I wrote is confusing, I can fix it if necessary. What I wanted to say is if you have people invited besides the organizer.

File: api-reference/beta/api/event_update.md (hunk 1 above)
Comment (baywet) [reply api-reference/beta/api/event_update.md]: Maybe the statement I wrote is confusing, I can fix it if necessary. What I wanted to say is if you have people invited besides the organizer

Comment (baywet) [reply api-reference/beta/api/event_update.md]: @angelgolfer-ms do you need me to update anything or are we good to go? Happy to help :)

**Comment by baywet** [comment]
@Lauragra any update on this PR?

**Comment by Lauragra** [comment]
@angelgolfer-ms , okay to merge this PR?

**Comment by baywet** [comment]
@Lauragra Anny chance we can merge this one as well soon?

**Comment by baywet** [comment]
Thanks!

**Pull request merged**